net-misc/openbgpd/openbgpd-9999.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools git-r3 systemd
+
+DESCRIPTION="OpenBGPD is a free implementation of BGPv4"
+HOMEPAGE="http://www.openbgpd.org/index.html"
+EGIT_REPO_URI="https://github.com/openbgpd-portable/openbgpd-portable.git"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	${DEPEND}
+	!!net-misc/quagga
+	acct-group/_bgpd
+	acct-user/_bgpd
+"
+BDEPEND="
+	dev-util/byacc
+	sys-devel/autoconf
```

Comment:
You don't need to depend on autoconf, automake usually and the eclass should cover this.

Reply:
removed